csfieldguide/static/interactives/number-memory/js/number-memory.js

```diff
@@ -0,0 +1,202 @@
+const STALL_TEXT = [
+  [1000, gettext("Finding some paper...")],
+  [300, gettext("*BONK*")],
+  [500, gettext("Ow!")],
+  [1000, gettext("Bumped my head.")],
+  [1500, gettext("Got it, now where was that pen?")],
+  [300, gettext("Ah ha!")],
+  [500, gettext("*Tap tap tap*")],
```

Comment:
Maybe this could be `*scribble scribble*` as well? I think it makes more sense to do the same action (as you do later on) to check if the pen is working :) Also maybe expand on `Empty. :(`, maybe even just `It's empty :(`.